### PR TITLE
Portfolio project

### PR DESCRIPTION
**Pull request by Haadiiii**
## Microverse portfolio Add contact form

 In this project we added:

- Added HTML file with text required.
- Added CSS style according to Figma template.
- Added readme file.
- Removed linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,250 +1,302 @@
...
+
+    <section>
+        
+            <div id="maincontact">
+                <div class="contact">
+                    <h2>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in
+                        touch.</h2>
+                </div>
+                <div class="form-contact">
+                    <form action="#">
```
Comment (Ismailco) [inline index.html:270]: - [ ] The form should contain the action with formspree, action value should be similar to `https://formspree.io/f/{form_id}`, kindly implement this so your form can be sent

Comment (Ismailco) [inline index.html:270]: - [ ] The form should contain a `POST` method, so the form can be submitted

File: style.css [hunk 2]
```diff
@@ -813,3 +830,120 @@ li {
...
+
+/* contact form start  */
+
+#maincontact {
+  position: absolute;
+  top: 6500px;
+  background-color: #fff;
+  display: grid;
+
+  /* padding: 24px 24px; */
```
Comment (Ismailco) [inline style.css:842]: - [ ] Kindly consider removing any commented unused code, it makes your code looks messy

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Haadiiii, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -1,250 +1,302 @@
...
+
+    <section>
+        
+            <div id="maincontact">
+                <div class="contact">
+                    <h2>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in
+                        touch.</h2>
+                </div>
+                <div class="form-contact">
+                    <form action="https://formspree.io/f/(form_id)" method="post">
```
Comment (codecaiine) [inline index.html:270]: - [ ] [REQUIRED] Good job so far, your form is almost working :+1:  but As the previous reviewer mentioned [here](https://github.com/Haadiiii/my-portfolio/pull/4#discussion_r914122155). Please go to [formspree service](https://formspree.io/html) and follow the different steps to configure and integrate into `action` a value similar to `https://formspree.io/f/{form_id}`.  You will be able to send information through your form to that platform. Now your form is not working. 

### Step One
Screenshot from 2022-07-05 19-38-22

### Step two
Screenshot from 2022-07-05 19-38-03

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Haadiiii , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

:heavy_check_mark: No linter errors
:heavy_check_mark: Used Github Flow
:heavy_check_mark: Work documented professionally
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Haadiiii) [reply index.html:270]: resolved 👍

**Review by isaacpitwa — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @Haadiiii  👋🏻 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**